### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by raminsalehi**

### CODE REVIEW

**Comment by markosankovic** [comment]
Build triggered. sha1 is merged.

**Comment by markosankovic** [comment]
Build started sha1 is merged.

**Comment by markosankovic** [comment]
Build finished.

File: examples/app_control_tuning/src/main.xc [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 /**
  * @file main.xc
- * @brief Demo application illustrates usage of module_pwm
+ * @brief Demo application illustrates a general example of tuning/controlling a somonet Drive.
```
Comment (romu4ld) [inline examples/app_control_tuning/src/main.xc]: typo "somonet"

File: examples/app_demo_general_pwm/doc/index.rst [hunk 2]
```diff
@@ -8,7 +8,7 @@ General PWM Application Demo
     :backlinks: none
     :depth: 3
 
-The purpose of this app (app_demo_general_pwm) is showing the use of pwm_service_general in :ref:`PWM Module <module_pwm>` in our different drives. For this purpose, a simple funciton (named send_pwm_values) is defined. This function changes the pwm values in each execution loop, and sends them to general pwm service through an interface. 
+The purpose of this app (app_demo_general_pwm) is showing the use of pwm_service_general in :ref:`PWM Module <module_pwm>` in different SOMANET Drives . For this purpose, a simple function (named send_pwm_values) is defined. This function changes the pwm values in each execution loop, and sends them to general pwm service through its corresponding interface.
```
Comment (romu4ld) [inline examples/app_demo_general_pwm/doc/index.rst]: typo space before dot "in different SOMANET Drives ."

**Comment by markosankovic** [comment]
Build triggered. sha1 is merged.

**Comment by markosankovic** [comment]
Build started sha1 is merged.

**Comment by markosankovic** [comment]
Build finished.

**Comment by markosankovic** [comment]
Build triggered. sha1 is merged.

**Comment by markosankovic** [comment]
Build started sha1 is merged.

**Comment by markosankovic** [comment]
Build finished.

**Pull request merged**